libs/milvus/tests/integration_tests/vectorstores/test_milvus.py

```diff
@@ -65,6 +65,22 @@ def test_milvus(temp_milvus_db: Any) -> None:
     assert_docs_equal_without_pk(output, [Document(page_content="foo")])
 
 
+def test_milvus_add_embeddings_search(temp_milvus_db: Any) -> None:
+    """Test end to end with add embeddings"""
+    embed_func = FakeEmbeddings()
+    docsearch = Milvus(
+        embed_func,
+        connection_args={"uri": temp_milvus_db},
+        drop_old=True,
+        consistency_level="Strong",
+        auto_id=True,
+    )
+
+    docsearch.add_embeddings(fake_texts, [embed_func.embed_documents(fake_texts)])  # type: ignore[list-item]
```

Comment:
maybe it's a little confusing when users add in only one dense embedding field. they may ask why it should be a list that contains only one element.
so i think in most cases(only one dense embedding field), this code should be `docsearch.add_embeddings(texts=fake_texts, embeddings=embed_func.embed_documents(fake_texts)) `, which keep len(texts) == len(embeddings)
so ideally, when there is only one embedding field, the embedding arg input can be like this:
```
[
    f1(a),
    f1(b),
    f1(c)
]
```
and when there are multiple embedding fields, the embedding arg input can be like this:
```
[
    [f1(a), f2(a)],
    [f1(b), f2(b)],
    [f1(c), f2(c)]
]
```
so the first axis is text number, the second axis is different embeding functions. besides, we have to judge whether the current Milvus is multi_vector. 
I will write some suggestion code below to implement this.

Reply:
Done with your suggestion